src/cryptonote_protocol/levin_notify.cpp

```diff
@@ -698,9 +776,31 @@ namespace levin
     }
     else
     {
-      zone_->strand.dispatch(fluff_notify{zone_, std::move(txs), source});
+      switch (tx_relay)
+      {
+        default:
+        case relay_method::none:
+        case relay_method::block:
+          return false;
+        case relay_method::stem:
+	  tx_relay = relay_method::fluff; // don't set stempool embargo when skipping to fluff
```

Comment:
Why does this do so without checking the embargo timer ?

Reply:
Looks like another tab snuck in here too or something. Will fix that.

This might require a bigger comment to explain. This is purely defensive programming. If someone requests stem mode over i2p/tor, the request cannot be done correctly because `on_transactions_relayed` (line 799) will put the tx in the stempool, making it eligible for broadcasting over ipv4/6 after an embargo timeout. So this was a quick hack so that "if I2P/Tor zone without white noise":

|  Requested Mode  |  New TxPool State  |
|-------------------------|--------------------------|
|           local             |            local             |
|           stem            |            fluff             |
|            fluff              |             fluff             |

Perhaps a `return false;` is better in this situation, indicating it can't work?

Also, if ipv4/6 local and stem requests map to Dandelion++ mode.

Reply:
Why would it be a problem to place an incoming tx in stem mode from i2p/tor in the txpool and fluff it after the embargo if not seen again ?

Reply:
A transaction received over i2p/tor is forwarded over public ipv4/6 in stem mode. The issue is the public interface allows a request to send transaction(s) over i2p/tor in stem mode, which should arguable be rejected (`return false`). Dandelion++ over i2p/tor requires mempool tracking the "zone" a transaction was received for the embargo timeout. I'm not aware of anything else blocking Dandelion++ over i2p/tor, but it might probably worth thinking about this mode separately. A sybil actor should already have difficulties when tracing origin IP over i2p/tor. However, Dandelion++ over i2p/tor can get interesting with white noise since it makes advanced ISP spying more difficult too.

Reply:
I don't understand, and I don't even know which part I don't understand here, but I'm happy enough given this is on purpose and not a mistake :)